app/views/layouts/header/_initial.html.erb

```diff
@@ -3,8 +3,8 @@
       aria-label="main navigation">
   <div class="container px-4">
     <div class="navbar-brand">
-      <%= link_to root_path, class: "navbar-item" do %>
-        <%= image_tag 'logo.svg', size: "34", class: 'for-mobile' %>
+      <%= link_to "https://get.budgetingkid.com", class: "navbar-item" do %>
```

Comment:
I don't think it's needed. Users should not be able to get to the landing page on mobile. Landing is needed only for SEO and selling the app in the web. Once they get to the app the landing becomes useless for them.